cli/lib/kontena/cli/services/services_helper.rb

```diff
@@ -283,6 +283,17 @@ def parse_service_id(service_id)
           end
         end
 
+        # Parses container name based on service name and instance number
+        # @param [String] service_name
+        # @param [Integer] instance_number
+        def parse_container_name(service_name, instance_number)
+          base = service_name.gsub('/', '-')
+          unless service_name.include?('/')
```

Comment:
That's what effectively happens with the `gsub` :) To change that to be more explicit requires other if/else branching which IMO in this case is not really required.